front/components/assistant_builder/types.ts

```diff
@@ -75,6 +75,12 @@ export type AssistantBuilderProcessConfiguration = {
   schema: ProcessSchemaPropertyType[];
 };
 
+// Websearch configuration
+
+export type AssistantBuilderWebsearchConfiguration = {
```

Comment:
Can we have a dedicated types file for websearch maybe?

Reply:
There is a dedicated file. 
But for those types specific to the assistant builder, it seems all `AssistantBuilder*Configuration` are defined there , so out of consistency I followed the pattern (see above in code). Is that ok with you?